apps/users/models.py

```diff
@@ -12,10 +14,12 @@ class User(Base):
     user_id = Column(INTEGER, primary_key=True, nullable=False)
     login = Column(VARCHAR(length=50), nullable=False, unique=True)
     password = Column(VARCHAR(length=50), nullable=False)
-    last_visit = Column(TIMESTAMP)
+    last_visit_at = Column(TIMESTAMP)
```

Comment:
why do u define column type as TIMESTAMP, but not DATETIME?

Reply:
Left it as it was before, and now it's working because column type is defined in main.sql. But of course the problem will raise after migrations appears. I will change it.